client/state/selectors/test/is-validating-account-recovery-key.js

```diff
@@ -0,0 +1,26 @@
+/**
+ * External dependencies
+ */
+import { assert } from 'chai';
+import deepFreeze from 'deep-freeze';
+
+/**
+ * Internal dependencies
+ */
+import { isValidatingAccountRecoveryKey } from '../';
+
+describe( 'isValidatingAccountRecoveryKey()', () => {
+	it( 'should return the requesting field under the validate substate tree.', () => {
```

Comment:
Oops. Thanks for the reminder, I can't count how many times I've forgotten that :)
Added by 6767b58